modules/navier_stokes/doc/content/source/fvkernels/PINSFVMomentumFriction.md

```diff
@@ -5,24 +5,16 @@ equations. This kernel must be used with the canonical PINSFV variable set,
 e.g. pressure and superficial velocity.  A variety of models are supported for
 the friction force:
 
-Linear friction, for laminar flow
-\begin{equation}
-F_i = - f v_i
-\end{equation}
-Quadratic friction, for turbulent flow
-\begin{equation}
-F_i = - f v_i |v_i|
-\end{equation}
 Darcy drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_i
+F_i = - \dfrac{f_i}{\epsilon} \rho v_d
 \end{equation}
 Forchheimer drag model
 \begin{equation}
-F_i = - \dfrac{f_i}{\epsilon} \rho v_i
+F_i = - \dfrac{f_i}{\epsilon} \rho v_d
 \end{equation}
 where $F_i$ is the i-th component of the friction force, f the friction factor, which may be anisotropic,
-$\epsilon$ the porosity and $\rho$ the fluid density and $v_i$ the fluid superficial velocity.
+$\epsilon$ the porosity and $\rho$ the fluid density and $v_d$ the fluid superficial velocity.
```

Comment:
I think that the $i$ index was meant to refer to the velocity component and you're retaining that on the force and coefficient. Seems like it might be worth keeping around on the velocity as well? Maybe $v_{d,i}$?

Reply:
ah good call. I ll fix it now